airflow/operators/__init__.py

```diff
@@ -76,6 +76,7 @@
         'TimeDeltaSensor',
         'TimeSensor',
         'WebHdfsSensor',
+        'BashSensor'
```

Comment:
This should be in the community-contributed operators section - rebase before adding, documentation structure has since changed.